tests/test_helm_upgrade_decision.py

```diff
@@ -16,24 +16,23 @@
 
 yaml = YAML(typ="safe", pure=True)
 case = TestCase()
+root_path = Path(__file__).parent.parent
 
 
 def test_get_all_cluster_yaml_files():
     expected_cluster_files = {
-        Path(os.getcwd()).joinpath("tests/test-clusters/cluster1/cluster.yaml"),
-        Path(os.getcwd()).joinpath("tests/test-clusters/cluster2/cluster.yaml"),
+        root_path.joinpath("tests/test-clusters/cluster1/cluster.yaml"),
+        root_path.joinpath("tests/test-clusters/cluster2/cluster.yaml"),
     }
 
-    result_cluster_files = get_all_cluster_yaml_files(is_test=True)
+    result_cluster_files = set(get_all_cluster_yaml_files(is_test=True))
```

Comment:
@sgibson91, the result it's turned into a set here, so probably this is why the generate job is not complaining. But I still believe we should update the comment to match what's happening currently or return the set directly

Reply:
That code link is from the test file, not the actual codebase, but probably something similar is happening in the codebase too (I'm finding this a difficult PR to review actually). But I agree, the comment should be updated now that I've checked that the logic isn't broken.

Reply:
huh! You're right, I didn't notice it's in a test